obj.js

```diff
@@ -272,7 +278,9 @@ async function mapOutgoingObjects(value) {
       for (let propName in obj) {
         json[propName] = await mapOutgoingObjects(obj[propName]);
       }
-      return json;
+      return {
+        json: json,
```

Comment:
I don't like the nested `json` here. Everything is JSON.

Reply:
Well you need some way of identifying this as containing a JSON object rather than being a description for a Function or a remote object...